server/api/accountRouter.ts

```diff
@@ -13,4 +14,36 @@ router.get("/", async (req: Request, res: Response, next: NextFunction) => {
   }
 });
 
+router.post("/", async (req: Request, res: Response, next: NextFunction): Promise<void>=> {
+  try {
+    const {accountType, accountName, institution, balance} : AccountAttributes = req.body
+    // const createAccount = 
+    await Account.create({
+      accountType,
+      accountName,
+      institution, 
+      balance
+    })
+    res.sendStatus(204)
```

Comment:
Should be code 200 right? 204 would imply success with no content.

Reply:
Or saying no need for content returned? I would think if a new account is created we'd need the UUID created to be returned for the user for use in redux.